content/blog/self_host_local_ai_assistant_with_jan_and_pinggy.md

```diff
@@ -0,0 +1,147 @@
+outputs:
+  - HTML
+  - AMP
+---
+{{< image "self_host_local_ai_assistant_with_jan_and_pinggy/jan_banner.webp" "Self-host Local AI Assistant with Jan and Pinggy" >}}
+
+Running your own AI assistant locally means keeping full control over your conversations and data while avoiding subscription fees and usage limits. {{< link href="https://jan.ai/" >}}Jan{{< /link >}} is an open-source ChatGPT alternative that runs entirely on your computer, powered by the robust Cortex inference engine. With [Pinggy](https://pinggy.io), you can also share your local Jan instance online for team collaboration or remote access.
+
+
+{{% tldr %}}
+
+1. **Download Jan**
+   - Install from <a href="https://jan.ai/" target="_blank">jan.ai</a> for Windows, Mac, or Linux
+   - No complex setup required - works out of the box
+
+2. **Get an AI Model**
+   ```
+   Recommended models:
+   • phi3.5 (for 8GB+ RAM)
+   • qwen2.5 (for older computers)
+   ```
+
+3. **Start Chatting**
+   - Use the New Chat tab for conversations
+   - Works 100% offline with full privacy
+
+4. **Share Online with Pinggy** 
+   ```bash
+   ssh -p 443 -R0:localhost:1337 -t qr@a.pinggy.io "u:Host:localhost:1337"
+   ```
+
+{{% /tldr %}}
+
+## Why Choose Jan for Your Local AI Assistant?
+
+Jan offers a compelling alternative to cloud-based AI services with several key advantages:
+
+- **Complete Privacy** - Your conversations never leave your computer
+- **No Subscription Fees** - Use powerful AI models without ongoing costs
+- **Offline Capability** - Works without internet connection
+- **User-Friendly Interface** - Familiar ChatGPT-like experience
+- **Open Source** - Fully transparent and customizable
+
+## Getting Jan Up and Running
+
+Getting started with Jan is surprisingly straightforward. Head over to {{< link href="https://jan.ai/" >}}jan.ai{{< /link >}} and grab the installer for your operating system. Whether you're on Windows, Mac, or Linux, there's a proper installer that handles everything for you.
```

Comment:
Jan is available for mac windows and linux. you can download and install pre built binaries from https://jan.ai/download

You can also build it from souce using:

```
git clone https://github.com/menloresearch/jan
cd jan
make dev
```

See detailed guide here: https://github.com/menloresearch/jan